versioned_docs/version-0.1.0/advanced-topics/messaging-system.md

```diff
@@ -33,4 +38,4 @@ A method turned into an RPC is no longer a regular method, it will have its own
 
 ## Network Update Loop
```

Comment:
Not relevant to this PR but as a side note. Network Update Loop is not really related to messaging so we might want to remove it from here.

Reply:
have removed the Network Update Loop